sys/riscv/trap.c

```diff
@@ -195,6 +140,14 @@ static bool fpu_handler(mcontext_t *uctx) {
   return true;
 }
 
+static vm_prot_t exc_access(u_long exc_code) {
+  if (exc_code == SCAUSE_INST_PAGE_FAULT)
+    return VM_PROT_EXEC;
+  else if (exc_code == SCAUSE_LOAD_PAGE_FAULT)
+    return VM_PROT_READ;
+  return VM_PROT_WRITE;
```

Comment:
`switch` statement is more readable imho.

Reply:
I agree. `switch` is less verbose in this case.